src/legacy/core_plugins/interpreter/init.ts

```diff
@@ -20,17 +20,28 @@
 // @ts-ignore
 import { register, registryFactory } from '@kbn/interpreter/common';
 
+// @ts-ignore
+import { Registry } from '@kbn/interpreter/common';
+
 // @ts-ignore
 import { routes } from './server/routes';
 
 import { Legacy } from '../../../../kibana';
 
 // @ts-ignore
-import { FunctionsRegistry } from './common/functions_registry';
-// @ts-ignore
-import { typeSpecs as types } from './common/types';
+import { typeSpecs as types } from '../../../plugins/data/common/interpreter/expression_types';
+
 // @ts-ignore
-import { TypesRegistry } from './common/types_registry';
+import { TypesRegistry } from './common';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { Fn } = require('@kbn/interpreter/common');
+
+export class FunctionsRegistry extends Registry<any, any> {
```

Comment:
you should import this one from `./common` the same you do with TypesRegistry

Reply:
See the latest refactorings.